widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -82,9 +82,13 @@ internal class CallView(
             ex?.run { showToast(this.debugMessage) }
         }
 
-        override fun onScreenSharingStarted() {
+        override fun onScreenSharingRequestSuccess() {
             Handler(Looper.getMainLooper()).post { appBar.showEndScreenSharingButton() }
         }
+
+        override fun onScreenSharingStarted() {
+            // no-op
+        }
```

Comment:
to avoid such kinds of `empty` functions I'd suggest making these functions inside the `interface ViewCallback` `default`, the such way it is possible to override only the functions we need.